vello_shaders/shader/fine.wgsl

```diff
@@ -987,18 +988,23 @@ fn main(
                 cmd_ix += 2u;
             }
             case CMD_BEGIN_CLIP: {
```

Comment:
My vague plan for implementing this was to turn it into a whole new command, as the operations are at least a little different. I presume you didn't want to do that to avoid the code duplication?